src/components/ForecastSummaries.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ForecastSummary from "./ForecastSummary";
+import "../styles/ForecastSummaries.css";
+
+function ForecastSummaries({ forecasts, onForecastSelect }) {
```

Comment:
Here I would definitely use object destructuring like this:
`const { date, description, icon, teperature } = forecast`
This would save you having to type `.forecast` every time